lib/peep_manager.rb

```diff
@@ -0,0 +1,32 @@
+require_relative './database_connect'
+require 'pg'
+
+class Peeps
+
+  extend DatabaseConnect
```

Comment:
I like that you've pushed your DatabaseConnect code into a class and then included it into your model classes in this way.